.azure-pipelines/function-tests.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-16.04'
+
+container: ninhu/r-sdk-build:latest
+
+steps:
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: |
+        pip install --user azureml-sdk
+        R -e 'dir.create(Sys.getenv("R_LIBS_USER"), recursive = TRUE);
+              .libPaths(Sys.getenv("R_LIBS_USER"));
+              # build and install R sdk
+              devtools::install_deps(upgrade = FALSE);
+              package_location <- devtools::build();
+              install.packages(package_location, repos = NULL)'
+    displayName: 'Build and Install SDK'
```

Comment:
in order to run testthat tests, do we need to build and install R sdk? just curious

Reply:
I guess tests won't work if you don't have azuremlsdk package pre-built.

Reply:
Got it.